TableViewDragger/TableViewDragger.swift

```diff
@@ -173,11 +175,22 @@ extension TableViewDragger {
 
             if let draggedCell = draggedCell(tableView, indexPath: dragIndexPath) {
                 let point = gesture.location(in: actualCell)
-                draggedCell.offset = point
-                draggedCell.transformToPoint(point)
-                draggedCell.location = gesture.location(in: tableView)
+                
+                if availableHorizontalScroll == true{
+                    
+                    draggedCell.offset = point
+                    draggedCell.transformToPoint(point)
+                    draggedCell.location = gesture.location(in: tableView)
+                    
+                } else {
+                    
+                    draggedCell.offset = CGPoint(x: (actualCell?.frame.size.width)!/2, y: point.y)
+                    draggedCell.transformToPoint(CGPoint(x: (actualCell?.frame.size.width)!/2, y: point.y))
+                    draggedCell.location = CGPoint(x: (actualCell?.frame.size.width)!/2, y: gesture.location(in: tableView).y)
```

Comment:
Should use `draggedCell` or `if let` because forced unwrapping of `actualCell` is dangerous.

Reply:
Got it.